zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -147,47 +188,40 @@ public void gracefullyShutdownConnections()
         clientConnectionsShutdown.gracefullyShutdownClientChannels();
     }
 
-    private ChannelFuture setupServerBootstrap(int port, ChannelInitializer channelInitializer)
-            throws InterruptedException
-    {
-        ServerBootstrap serverBootstrap = new ServerBootstrap().group(
-                serverGroup.clientToProxyBossPool,
-                serverGroup.clientToProxyWorkerPool);
+    private ChannelFuture setupServerBootstrap(
+            SocketAddress listenAddress, ChannelInitializer<?> channelInitializer) throws InterruptedException {
+        ServerBootstrap serverBootstrap =
+                new ServerBootstrap().group(serverGroup.clientToProxyBossPool, serverGroup.clientToProxyWorkerPool);
 
         // Choose socket options.
         Map<ChannelOption, Object> channelOptions = new HashMap<>();
         channelOptions.put(ChannelOption.SO_BACKLOG, 128);
-        //channelOptions.put(ChannelOption.SO_TIMEOUT, SERVER_SOCKET_TIMEOUT.get());
         channelOptions.put(ChannelOption.SO_LINGER, -1);
         channelOptions.put(ChannelOption.TCP_NODELAY, true);
         channelOptions.put(ChannelOption.SO_KEEPALIVE, true);
 
-        // Choose EPoll or NIO.
-        if (USE_EPOLL.get()) {
-            LOG.warn("Proxy listening with TCP transport using EPOLL");
-            serverBootstrap = serverBootstrap.channel(EpollServerSocketChannel.class);
-            channelOptions.put(EpollChannelOption.TCP_DEFER_ACCEPT, Integer.valueOf(-1));
-        }
-        else {
-            LOG.warn("Proxy listening with TCP transport using NIO");
-            serverBootstrap = serverBootstrap.channel(NioServerSocketChannel.class);
-        }
+        LOG.debug("Proxy listening with " + serverGroup.channelType);
```

Comment:
This should be `INFO`.

Reply:
See comment above.   I'd say this isn't actionable, and should be exposed via a programmatic api up to the caller.

Reply:
Yeah I'd change this to `INFO` (and the other logging of port binding too). It's true to say its not actionable and therefore shouldn't be `WARN`, but I have surprisingly frequently looked at these particular statements in logs to verify that configurations of ports have been applied as intended, so I'd keep them above `DEBUG`.